repeater.js

```diff
@@ -34,6 +34,11 @@ const args = require('yargs')
     type: 'string',
     requiresArg: false
   })
+  .option('vmssList', {
+    describe: 'Get the list of names of required VMSS Scale Sets',
+    type: 'array',
+    requiresArg: false
+  })
```

Comment:
Requiring the names of all VMSSs to be known at startup is too limiting -- at the time the repeater is started the name of another VMSS isn't known.  

Instead could this use Azure's tags?  I assume there's a way to use their API to find resources (in this case VMSSs) by tag.